plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/ApplicationQuickAssistProcessor.java

```diff
@@ -17,7 +17,7 @@
 package com.google.cloud.tools.eclipse.appengine.validation;
 
 /**
- * Provides quick assists for appengine-web.xml in the source editor.
+ * Provides remove application quick fix for appengine-web.xml in the source editor.
```

Comment:
I think "application" could be confusing to someone new. How about `Provides quick fixes to remove the {@code application} element from ...` as you did with `VersionQuickAssistProcessor` below. (BTW, is it correct that `fixes` are to be plural?)

Reply:
done
singular is probably slightly more correct here.